src/main/java/gregtech/api/items/metaitem/stats/IDataStick.java

```diff
@@ -0,0 +1,4 @@
+package gregtech.api.items.metaitem.stats;
+
+public interface IDataStick {
```

Comment:
In game, the data orb says that it can hold more entries than the data stick (in the tooltip), might this be something to incorporate into this interface, as currently I do not think this is implemented.

Reply:
Right now data items only store one research key on their NBT tags. I've allowed the data item to be customizable for each autogenerated research recipe, so we could use the data orb for specific recipes instead of the stick to simulate this instead. 

Implementing multiple keys on the data item is currently not very feasible - as recipes do not allow input-aware outputs. This means we cannot output the data stick with the input's original NBT and a new key appended to it.

Reply:
If that is the case, then please look at adjusting the tooltips of the data orb to not imply that it can store more recipes